# RNASeq-M.tuberculosis
A simple step-wise script for analysis of M. tuberculosis RNASeq data. 

Dr. Nikhil Bhalla optimised this RNA-seq pipeline under the supervision of Dr. Ranjan Nanda at ICGEB, New Delhi. 


How to execute the analysis?
Step one: Bring all Paired-end Fastq files to a folder. The filenames should have a common prefix if they are Forward and Reverse reads of the same sample and end with "1.fastq.gz". Make sure the read files are gzipped. If not, use the "pigz *.fastq" command to compress. 
Step two: Wherever the script folder is unarchived, you can execute the same scripts from anywhere on the system. For example, if the Script folder is stored on the Desktop and you have you read files in "C/New_folder". You can to execute the scripts one by one in same folder. 
Step three: Use IGEAK! 2021 version or IDEP2.0 for subsequent analysis
s. 
In case of queries, contact: nikhilbhalla94@gmail.com
