# RNASeq-M.tuberculosis
A simple step wise script for analysis of M. tuberculosis RNASeq data. 

This RNA-seq pipeline was optimized by Dr. Nikhil Bhalla under the supervision of Dr. Ranjan Nanda.


How to execute the analysis?
Step one: Bring all Paired-end fastq files in a folder. The filenames should have a common prefix if they are Forward and Reverse reads of same sample and their should end with "_1.fastq.gz". Make sure the read files are g-zipped. If not, use "pigz *.fastq" command to compress. 
Step two: Where ever the script folder is unarchived, you can execute the same scripts from anywhere on the system. for example if the Script folder is stored on Desktop and you have you read files in "C/New_folder". You need to execute the scripts one by one in same folder. 
Step three: Use IGEAK! 2021 version for subsequent analysis. 

In case of queries, contact: nikhilbhalla94@gmail.com
